src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -967,6 +851,122 @@ private void ParseExecutionPolicy(string[] args, ref int i, ref string execution
             executionPolicy = args[i];
         }
 
+        private bool ParseFile(string[] args, ref int i, bool noexitSeen)
+        {
```

Comment:
Making ParseFile a local function would make it inconsistent with similar methods like ParseCommand(), ParseFormat(), ParseExecutionPolicy(), etc...